docs/blog/2019-03-27-security-concerns-unique-to-static-sites/index.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Security for static sites.
+date: 2019-03-27
+author: Alex Moon
+excerpt: "While more secure than their server side counterparts, static sites provide new and unique security concerns."
+tags:
+  - security
+  - api
+  - JAMstack
+---
+
+Among the many lauded benefits of using a static site generator (i.e. Gatsby) is security. So much so, that in the Gatsby blog there has never been a single post focussed on mitigating security issues. There is not even a doc! This is troubling, because, it creates the myth that there are no security issues to worry about. HINT: You should still be worried about security.
+
+Please understand, there are whole classes of security vulnerabilities that are eliminated by using any static site. But, there are still others that are shared and some that are created. You just need to make sure you're aware of those new concerns and how to remedy them. But, What are they and how do you avoid them?
+
+This article will briefly cover shared security concerns that exist between all flavors of website. It will then take dive deeper on those security concerns unique to static sites.
+
+## Defining Terms
+
+These are not to be considered universal definitions, just how the author will use them going forward.
+
+- **site** - Any website, web-app, or other content you can browse on the internet in a web browser.
+
+- **server-side site** - Any site that renders dynamic content on the server, at time of request -- whether using node.js, PHP, Python, or any other method -- before delivering it to the user's browser.
+
+- **client-side site** - Gatsby only generates static sites. Single Page Apps (SPAs) are different but any security issue affecting a pure static site will also affect an SPA. Collectively both methods deliver all needed assets(HTML/CSS/JS) to the client for site rendering.
+
+- **attack vector** - A resource through which to hack a site. Just because an attack vector exists doesn't mean you site is vulnerable to that attack. A server is an attack vector. But, the server requires a vulnerability for an attacker to exploit(use) the server as an attack vector.
+
+## Shared Security Concerns
+
+No matter how you deploy your site, how little javascript you use, whether it is sever-side or client-side: your site has attack vectors.
+
+When it comes to static sites, if you are going to get hacked, it is going to be at the infastructure level (server/CDN, DNS, CI, etc.) or network level([MitM](https://en.wikipedia.org/wiki/Man-in-the-middle_attack)). Thanks to the cloud, most of this work is done for you and you do not have to secure these layers. As long as you are using two-factor authentication(2FA) and a strong password. You are all using 2FA (when possible) and strong passwords, right!? Good.
```

Comment:
```suggestion
When it comes to Gatsby sites, if you are going to get hacked, it is likely going to be at the infrastructure level (server/CDN, DNS, CI, etc.) or network level([MitM](https://en.wikipedia.org/wiki/Man-in-the-middle_attack)). Thanks to the cloud, most of the work is done for you and you do not have to secure these layers. As long as you are using two-factor authentication (2FA) and a strong password. You are all using 2FA (when possible) and strong passwords, right!? Good.
```

Reply:
This seems like a good place to mention HTTPS as well. Ideally hosting platforms will take care of this for you, but there is always Let's Encrypt for those that don't.